### ISSUE TITLE
API Key not set

### ISSUE THREAD

**Issue by MrAshRhodes** [guywaldman/magic-cli#29]
Howdy, im probably missing something real simple.

I updated the config file to have my openai api key, when running commands it says this isnt set 

ive also done `magic-cli config set -k openai.api_key -v sk-proj-ABC`

When i list the config it shows the key has a value. 
![image](https://github.com/user-attachments/assets/74caa8de-a4ad-4cf5-b504-618579521bec)

Sys Info 

```
❯ magic-cli sys-info
System information as detected by the CLI:

OS: Darwin
OS version: 15.0
CPU architecture: arm64
Shell: zsh
```
The config file exists, i can see it and it has all the settings set.

There's gonna be something simple im missing right...

**Comment by thiswillbeyourgithub** [comment]
Same here

**Comment by guywaldman** [comment]
Hi there, taking a look.

**Comment by guywaldman** [comment]
@MrAshRhodes @thiswillbeyourgithub Fixed in release 0.0.4 (being pushed out now).
Apologies for the inconvenience, added test coverage to avoid such regressions in the future.
